requirements.txt

```diff
@@ -1,2 +1,3 @@
+Genshi
```

Comment:
Yes.  It's a templating language.  We use it in this instance to parse the XML in an event-driven manner.  It was used over more traditional xml libraries because it will actually tell us what the XML comments are.